cs-amazon/src/main/java/io/cloudslang/content/amazon/actions/loadbalancing/CreateLoadBalancer.java

```diff
@@ -0,0 +1,185 @@
+package io.cloudslang.content.amazon.actions.loadbalancing;
+
+import com.hp.oo.sdk.content.annotations.Action;
+import com.hp.oo.sdk.content.annotations.Output;
+import com.hp.oo.sdk.content.annotations.Param;
+import com.hp.oo.sdk.content.annotations.Response;
+import com.hp.oo.sdk.content.plugin.ActionMetadata.MatchType;
+import com.hp.oo.sdk.content.plugin.ActionMetadata.ResponseType;
+import io.cloudslang.content.amazon.entities.constants.Outputs;
```

Comment:
We should use the constants from CloudSlang commons: io.cloudslang.content.constants.OutputNames